### DISCUSSION TITLE
Add formatter/prettier/eslint

### DISCUSSION THREAD

**Discussion by YannHulot** [Etheonor/supanextail#3] [Feedback]
Consider adding a de facto formatter such as prettier used with either eslint/tslint.
The last thing you want to worry about when starting a new project is linting and formatting. 

If everything is set up properly, you could even add vs code setting file that triggers the auto formatting on save. But either way, even if you don't do that, you can still have prettier  and eslint setup with some basic rules ready to go.

**Comment by Etheonor** [comment]
Done ! (ESlint + prettier)
